cookbook/specs/fvm.json

```diff
@@ -1831,21 +2833,165 @@
                             },
                             {
                               "name": "pruning",
+                              "parse_directive": {
+                                "function_tag": "GET_BLOCK_BY_NUM"
+                                },
                               "values": [
                                 {
-                                  "expected_value": ""
+                                  "latest_distance": 2800
                                 },
                                 {
                                   "extension": "archive",
-                                  "expected_value": ""
+                                  "expected_value": "1"
                                 }
                               ]
                             }
+                        ],
+                        "extensions": [
+                            {
+                                "name": "archive",
+                                "cu_multiplier": 5,
+                                "rule": {
+                                    "block": 2600
```

Comment:
this should be 23H and 40 min:
Blocks20M = 20 * 60 / 30 = 40
Block23H40M = Blocks24H - Blocks20M = 2840

Reply:
done